js/screens/welcome-screen.js

```diff
@@ -38,22 +38,34 @@ ultimatePixel();
 championships();
 
 function lineBelowUPC(orangeLine) {
-   const orangeLine = document.createElement('div');
-   orangeLine = src('./components/orangeLine.svg');
-   return orangeLine;
+   const svgLine = 'assets/orangeLine.svg';
+   const svgL = document.createElementNS(svgLine, 'svg');
+   document.body.append(svgL);
+   return svgLine;
 }
 
-function margins(marginHeader, marginFooter) {
-   const marginHeader = document.createElement('div');
-   marginHeader = src('./components/marginHeader.svg');
-   const marginFooter = document.createElement('div');
-   marginFooter = src('./components/marginFooter.svg');
-   return marginNav && marginFooter;
+const svgTopMargin = 'assets/marginTop.svg';
+const svgBottomMargin = 'assets/marginBottom.svg';
+
+function margins() {
+   const svgMarginTop = document.createElementNS(svgTopMargin, 'svg');
+   const svgMarginBottom = document.createElementNS(svgBottomMargin, 'svg');
+   document.body.append(svgMarginTop && svgMarginBottom);
+   return svgMarginTop && svgMarginBottom;
 }
 
-function addSwords() {}
+const svgLeftSword = 'assets/leftSword.svg';
+const svgRightSword = 'assets/rightSword.svg';
+
+function addSwords() {
```

Comment:
**nitpick:** These changes are not part of this pull request. You can fix it by doin `git pull origin main` on your current branch